helphours/account_routes.py

```diff
@@ -19,8 +19,10 @@ def login():
                 message = "Incorrect email or password"
             elif not user.is_active:
                 message = "This account is inactive"
+                log.info(f'{user.first_name} attempted to log in, but account was deactivated.')
```

Comment:
Include last name also to bring into consistency with else, which logs first and last name?